packages/desktop-electron/index.ts

```diff
@@ -60,9 +60,21 @@ function createBackgroundProcess() {
   serverProcess = utilityProcess.fork(
     __dirname + '/server.js',
     ['--subprocess', app.getVersion()],
-    isDev ? { execArgv: ['--inspect'] } : undefined,
+    isDev ? { execArgv: ['--inspect'], stdio: 'pipe' } : { stdio: 'pipe' },
```

Comment:
Piping out from the forked server process - allows us to handle the stdout/stderr